### DISCUSSION TITLE
Gauge does not Light up

### DISCUSSION THREAD

**Discussion by polyefko** [flybywiresim/aircraft#4125] [Help]
Crazy question...All of a sudden the gauge that shows flight path is not lighting up. I turn the bright/dim knob and it does not brighten that gauge. Its totally dark. All other knobs work fine.
Forgot the name of the gauge but its the second screen from the left......(my bad).

**Comment by polyefko** [comment]
To clarify its the ND gauge that is total dark

**Comment by tracernz** [comment]
Try with no other mods in your community directory to see if the issue persists.
